src/main/resources/db/changesets/20200625_initial_schema.yaml

```diff
@@ -50,18 +50,25 @@ databaseChangeLog:
               name: wait_until
               type: timestamp
               constraints:
-                nullable: false
+                nullable: true
                 unique: false
     - createTable:
         tableName: label
         columns:
+          - column:
+              name: id
+              type: uuid
+              constraints:
+                primaryKey: true
+                unique: true
+                nullable: false
```

Comment:
Why do we need a primary unique id for labels?

Reply:
(1) A tracked resource will have multiple labels, and I am not sure a resource can have different values for the same key. 
(2) When handling duplicate labels, how we want to handle labels inside? Create new one or upgrade the existing ones?

If key + tracked_rource_id are enough, we can use those two as PK and remove this uuid?

Reply:
I think key + tracked_resource_id are enough for now. I could see us being in a world where we allow multiple values for the same key.
I don't think we're inserting new labels in many if all of our APIs. Let's handle the semantics of that when we get there.

Reply:
okay! Also remove the state, because that should join the main table if we want that value.